src/views/product-landing/components/product-landing-blocks/blocks/collection-cards/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { CollectionCardsProps } from './types'
+import s from './collection-cards.module.css'
+
+function CollectionCards({ collectionCards }: CollectionCardsProps) {
```

Comment:
Another placeholder for `CollectionCards`, similar to the placeholders in #390 .